src/EnergyPlus/WindowManager.cc

```diff
@@ -4591,7 +4572,7 @@ namespace Window {
 
     //******************************************************************************
 
-    void WindowGasPropertiesAtTemp(EnergyPlusData &state,
+    void WindowGasPropertiesAtTemp(EnergyPlusData const &state,
```

Comment:
[src/EnergyPlus/WindowManager.cc:4594]:(style),[constParameterReference],Parameter 'state' can be declared as reference to const